renderdoc/driver/vulkan/vk_posix.cpp

```diff
@@ -55,6 +55,17 @@ bool WrappedVulkan::AddRequiredExtensions(bool instance, vector<string> &extensi
       extensionList.push_back(VK_KHR_SURFACE_EXTENSION_NAME);
 
     bool oneSurfaceTypeSupported = false;
+ 
+#if defined(VK_USE_PLATFORM_ANDROID_KHR)
```

Comment:
Is this maybe logical double-add merge conflict, with the code I added recently down on line 106?